packages/react-composites/src/composites/CallComposite/components/ComplianceBanner.tsx

```diff
@@ -56,10 +57,10 @@ export const ComplianceBanner = (props: ComplianceBannerProps): JSX.Element => {
   return <DismissableMessageBar variant={variant} strings={strings} />;
 };
 
-function DismissableMessageBar(props: { variant: number; strings: CompositeStrings }): JSX.Element {
+function DismissableMessageBar(props: { variant: string; strings: CompositeStrings }): JSX.Element {
```

Comment:
can we strongly type the variant everywhere instead of `string`?
e.g.
```ts
type ComplianceBannerVariant =
  | 'NO_STATE'
  | 'TRANSCRIPTION_STOPPED_STILL_RECORDING'
  | 'RECORDING_STOPPED_STILL_TRANSCRIBING' ...etc.
```

Reply:
Ok np!